Utils/DrippyBlockClient.ts

```diff
@@ -0,0 +1,15 @@
+import { Client, ClientOptions } from "minecraft-protocol";
+import { PlayerManager } from "../Minecraft/PlayerManager";
+import * as mc from 'minecraft-protocol';
+
+
+export class DrippyBlockClient extends Client {
```

Comment:
Please change this name to something along the lines of `MinecraftBotClient`

Reply:
Ok :(